content/rs/references/client_references/client_ioredis.md

```diff
@@ -0,0 +1,143 @@
+---
+Title: Using Redis with Node.js (ioredis)
+description:
+weight:
+alwaysopen: false
+categories: ["RS"]
+---
+In order to use Redis with Node.js, you need to install a Node.js Redis client.
+In the following sections, we demonstrate the use of [ioredis](https://github.com/luin/ioredis),
+a community recommended Redis client for Node.js with support for Promises built in.
+The other community recommended client for Node.js developers is [node_redis]({{< relref "./client_nodejs" >}}).
+
+Additional Node.js clients for Redis can be found under the [Node.js section](https://redis.io/clients#Node.js) of the [Redis Clients page](https://redis.io/clients).
+
+## Installing ioredis
+
+Complete instructions for installing ioredis can be found in the [README file](https://github.com/luin/ioredis/blob/master/README.md).
+To install ioredis, run:
+
+    npm install ioredis 
+
+## Connecting to Redis
+
+This example code creates a connection to Redis:
+
+    ```sh
+        const Redis = require('ioredis');
+        const redis = new Redis({
+        host: 'hostname',
```

Comment:
ident the code

Reply:
Fixed, including in a couple other places where I've standardized indents, and removed other `sh` blocks making them `javascript` instead.